### DISCUSSION TITLE
undici network problems by using discord.js v14

### DISCUSSION THREAD

**Discussion by reflowify-enricog** [nodejs/undici#1617] [Q&A] [answered]
Hello dear undici team. I have the problem for days that I can't start my newly written discord.js v14 bots on my Raspberry Pi, although they work fine on my Pc. Why do I come to you ask yourself? Your package is giving me an error that I can't stand anymore. Every time I start my Discord bot and it tries to create a server using your package, it crashs with a 'Connect Timeout Error'. I thought it was my internet at first, but as of today I'm using a lan cable, so my Raspberry Pi is doing 50mb/s download and 12mb/s upload, as well as my home pc, where all bots are working fine. Why do I still get this error message? The file path where this error comes from is: '/node_modules/undici/lib/core/connect.js:108:24'. 
Please help me.

Kind regards Enderzombie

**Comment by reflowify-enricog** [comment, marked as answer]
Problem solved

**Comment by KhafraDev** [comment]
Duplicate of https://github.com/nodejs/undici/issues/1531 (solution: https://github.com/nodejs/undici/issues/1531#issuecomment-1178869993)
